src/apps/patron-page/PatronPage.tsx

```diff
@@ -110,61 +107,56 @@ const PatronPage: FC = () => {
-              showCheckboxes
-            />
-          )}
-          <StatusSection />
-          {patron && (
-            <ReservationDetailsSection
-              changePatron={changePatron}
-              patron={patron}
-            />
-          )}
-          {patron && <PincodeSection changePincode={setPin} required={false} />}
-          {successPinMessage && (
-            <p className="text-body-small-regular mb-8 mt-8" role="alert">
-              {successPinMessage}
-            </p>
-          )}
+    <form className="dpl-patron-page" onSubmit={(e) => handleSubmit(e)}>
+      <h1 className="text-header-h1 my-32">{t("patronPageHeaderText")}</h1>
+      <NotificationComponent />
+      {patron && <BasicDetailsSection patron={patron} />}
+      <div className="patron-page-info">
+        {patron && (
+          <ContactInfoSection
+            changePatron={changePatron}
+            patron={patron}
+            inLine={false}
+            showCheckboxes
+          />
+        )}
+        <StatusSection />
+        {patron && (
+          <ReservationDetailsSection
+            changePatron={changePatron}
+            patron={patron}
+          />
+        )}
+        {patron && <PincodeSection changePincode={setPin} required={false} />}
+        {successPinMessage && (
+          <p className="text-body-small-regular mb-8 mt-8" role="alert">
+            {successPinMessage}
+          </p>
+        )}
 
-          <button
-            data-cy="save-user-patron"
-            className="mt-48 btn-primary btn-filled btn-small arrow__hover--right-small "
-            type="submit"
-            disabled={disableSubmitButton}
-          >
-            {disableSubmitButton
-              ? t("patronPageLoadingText")
-              : t("patronPageSaveButtonText")}
-          </button>
+        <button
+          data-cy="save-user-patron"
+          className="mt-48 btn-primary btn-filled btn-small arrow__hover--right-small "
+          type="submit"
+          disabled={disableSubmitButton}
+        >
+          {disableSubmitButton
+            ? t("patronPageLoadingText")
+            : t("patronPageSaveButtonText")}
+        </button>
 
-          <div className="text-body-small-regular mt-32">
-            {t("patronPageDeleteProfileText")}{" "}
-            <Link
-              id="delete-patron-link"
-              href={deletePatronUrl}
-              className="link-tag"
-            >
-              {t("patronPageDeleteProfileLinkText")}
-            </Link>
-          </div>
+        <div className="text-body-small-regular mt-32">
+          {t("patronPageDeleteProfileText")}{" "}
+          <Link
+            id="delete-patron-link"
+            href={deletePatronUrl}
+            className="link-tag"
+          >
+            {t("patronPageDeleteProfileLinkText")}
+          </Link>
         </div>
-      </form>
-      {patron && (
-        <PauseReservation user={patron} id={pauseReservation as string} />
```

Comment:
What about the `PauseReservation` component? Shouldn't it be removed?

Reply:
Great question! Thanks for asking 🥇 
I checked though, and it is still in use in the reservation list component:
![image](https://github.com/danskernesdigitalebibliotek/dpl-react/assets/28546954/cca5219f-d33b-4a6c-8179-197b7206e34b)